src/lib/Types/Core.hs

```diff
@@ -564,16 +565,27 @@ instance SubstE AtomSubstVal EffectOpType
 deriving instance Show (EffectOpType n)
 deriving via WrapE EffectOpType n instance Generic (EffectOpType n)
 
+type AbsDict = Abs (Nest Binder) Dict
+
 data SpecializedDictDef n =
-  -- dict type abstracted over local params, method names
-  SpecializedDictDef (Abs (Nest Binder) DictType n) [AtomName n]
-  deriving (Show, Generic)
+   SpecializedDict
+     -- Dict is abstracted over local "data" params. The methods are
```

Comment:
This comment ends in the middle of a sentence?

Reply:
I also don't like calling them "local" data params. There's nothing local here, this is at the top level. It's just abstracted over some data params we chose at the abstraction site, but they didn't have to be local at all (e.g. the 10 in `Fin 10`)

Reply:
Agreed. Are we happy with calling them just "data params" then?

Reply:
Or "runtime params"?